frontend/src/app/main/refs.cljs

```diff
@@ -183,6 +183,12 @@
 (def context-menu
   (l/derived :context-menu workspace-local))
 
+(def file-library-listing-thumbs?
+  (l/derived :file-library-listing-thumbs? workspace-global))
```

Comment:
Here, the var name it is ok to be with `?`, but the state access should be changed accordingly to the previous change

Reply:
Fixed